spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-client-reactive-discovery-client-it/src/test/resources/spring-cloud-kubernetes-client-reactive-discovery-it-service.yaml

```diff
@@ -1,14 +0,0 @@
-apiVersion: v1
```

Comment:
I removed this project, entirely. There is already such a project that does the same exact tests... I do not know (or remember) what happened and how we ended up in this situation, but this needs to be cleaned up.